__tests__/app.test.js

```diff
@@ -273,6 +281,7 @@ describe('Test POST /api/article/:article_id/comments endpoint', () => {
   })
 })
```

Comment:
Please add one more test with status 400, where the 'request object' is missing required field(s), e.g. no username or body properties.